optaweb-vehicle-routing-backend/pom.xml

```diff
@@ -105,6 +115,10 @@
           <groupId>org.hamcrest</groupId>
           <artifactId>hamcrest</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>javax.xml.bind</groupId>
+          <artifactId>jaxb-api</artifactId>
+        </exclusion>
```

Comment:
This doesn't seem to be necessary.

Reply:
(Minor suggestion. Doesn't block the PR.)

Reply:
This exclusion is necessary because there were duplicate classes between this dependency and `jakarta.xml.bind-api`, which broke the build locally from `mvn clean install -Dfull`. Not sure if this is expected behaviour since `jakarta.xml.bind-api` is `runtime` scoped.

Reply:
I don't think scope affects this.

https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-test/2.1.12.RELEASE doesn't depend on `jaxb-api` so I'm still convinced this exclusion is redundant (and my local build confirms that).

This a trivial issue. The `7.33.x` branch is effectively dead for this project, so no need for perfection. Feel free to ignore this.

Reply:
@yurloc I can't confirm this locally, enforcer repeatedly fails without this exclusion :/

To be on the safe side I'll keep this exclusion. But thanks for confirming on your local build.

Reply:
Yes, sure. No problem!